webroot/adm/oncall/generate-cstg-keypair.html

```diff
@@ -0,0 +1,650 @@
+                            <h5>Search Results:</h5>
+                        </div>
+                    </div>
+                    <div class="row pt-3 px-2 collapse" id="searchSiteNoSiteFoundAlert">
+                        <div class="alert alert-warning d-flex align-items-center pt-3" role="alert">
+                            <svg class="bi flex-shrink-0 me-2" width="24" height="24" role="img" aria-label="Warning:">
+                                <use xlink:href="#exclamation-triangle-fill" />
+                            </svg>
+                            <div>
+                                No site found
+                            </div>
+                        </div>
+                    </div>
+                    <div class="row pt-3 px-2 collapse" id="searchSiteErrorAlert">
+                        <div class="alert alert-danger d-flex align-items-center pt-3" role="alert">
+                            <svg class="bi flex-shrink-0 me-2" width="24" height="24" role="img" aria-label="Warning:">
+                                <use xlink:href="#exclamation-triangle-fill" />
+                            </svg>
+                            <div id="searchOutputErrorDiv">
+                            </div>
+                        </div>
+                    </div>
+                    <div class="row px-2 table-responsive">
+                        <table class="table table-bordered" id="searchResultsTable">
+                            <thead>
+                                <th scope="col" class="col-1">Sites</th>
+                                <th scope="col" class="col-1">CSTG Keypairs</th>
+                            </thead>
+                            <tbody id="searchResultsTableBody">
+                            </tbody>
+                        </table>
+                    </div>
+                </div>
+            </div>
+            <div class="row">
+                <div class="card card-body">
+                    <div class="row pt-1">
+                        <div class="col">
+                            <div class="row">
+                                <h5>Create / Update Operations:</h5>
+                            </div>
+                            <div class="row py-2">
+                                <div>
+                                    Use this to create new site, update existing sites or create new key pairs.
+                                </div>
+                            </div>
+                            <div class="row">
+                                <div class="col">
+                                    <button type="button" aria-controls="createSiteRegion" aria-expanded="false"
+                                        id="createSite" data-bs-parent="#siteParent" class="btn btn-secondary group"
+                                        data-bs-toggle="collapse" data-bs-target="#createSiteRegion"
+                                        role="button">Create site</button>
```

Comment:
nit: `<button>` already as the implicit role of `"button"`, so we don't need to specify it explicitly. There's a few instances in this file and in `generate-api-key-secrets.html`